# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/ffvpx/config_override.h
# Commit: 9973c75413d6
# Full Hash: 9973c75413d6237b9005b9ab908898a153bd2f06
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-02-28 04:30:28
# Regressor Bug: 1879873
# File Overlap Count: 1
# Description:
#   Bug 1879873 - Remove MOZ_LIBAV_FFT. r=karlt
#   
#   We have some time to use av_fft.h, after which we'll have to use tx.h directly.
#   It's not hard to update.
#   
# ==============================================================================

diff -r 7c947114da80 -r 9973c75413d6 media/ffvpx/config_override.h
--- a/media/ffvpx/config_override.h	Tue Feb 27 12:40:22 2024 +0000
+++ b/media/ffvpx/config_override.h	Tue Feb 27 12:40:22 2024 +0000
@@ -1,11 +1,9 @@
 // This file contains overrides for config.h, that can be platform-specific.
 
-#ifdef MOZ_LIBAV_FFT
 #undef CONFIG_FFT
 #undef CONFIG_RDFT
 #define CONFIG_FFT 1
 #define CONFIG_RDFT 1
-#endif
 
 // override '#define EXTERN_ASM _' in config_generic.h to allow building with
 // gcc on arm